tests/SIL.LCModel.Tests/DomainServices/WritingSystemServicesTests.cs

```diff
@@ -255,6 +255,34 @@ public void UpdateWritingSystemListField_RemovesMergedCodeAfterMergeWith()
 
 		/// <summary/>
 		[Test]
+		public void DeleteWritingSystem()
+		{
+			int m_wsEn = Cache.WritingSystemFactory.GetWsFromStr("en");
+			int m_wsFr = Cache.WritingSystemFactory.GetWsFromStr("fr");
+
+			CoreWritingSystemDefinition enBlz;
+			WritingSystemServices.FindOrCreateWritingSystem(Cache, null, "blz", false, false, out enBlz);
+
+			var revIndex = Cache.ServiceLocator.GetInstance<IReversalIndexRepository>().FindOrCreateIndexForWs(m_wsEn);
+
```

Comment:
I think we want to add the revIndex into the LexDb ReversalIndexesOC collection here. Unless FindOrCreateIndexForWs already does that.

Reply:
I added LexDb ReversalIndexesOC collection.